README.md

```diff
@@ -1,2 +1,133 @@
 # components-java-sdk
-Dapr SDK to compose pluggable components in Java.
+
+[Dapr] SDK to compose Pluggable components in Java. Also defines some sample components.
```

Comment:
nit:
I guess it's missing a link there
```suggestion
[Dapr](https://dapr.io/) SDK to compose Pluggable components in Java. Also defines some sample components.
```

Reply:
No, this is a reference link. That reference is defined at the end of the document. 
 https://daringfireball.net/projects/markdown/syntax#link and https://www.markdownguide.org/basic-syntax/#reference-style-links